src/components/UserAvatar.vue

```diff
@@ -21,7 +21,7 @@
     >
       <a
         v-if="user.displayName"
-        :href="`${LIKER_LAND_URL_BASE}/${user.id}`"
+        :href="`${LIKER_LAND_URL_BASE}/${user.id}?utm_source=button`"
```

Comment:
should we explicitly include `avatar` in utm_source?

Reply:
I think avatar is the only component that links to this